docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>I would either add more detailed description or a link for each speacial URL. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>resolved</p>